### ISSUE TITLE
Sending photos and audio

### ISSUE THREAD

**Issue by Principe7** [mattford/scoutlink#35]
Hello @mattford 

Ask if you can add the option to send photos in the application, that is, a kind of logo or icon of a camera appears and users can image the files on their mobile device.

The same if you can also add the option to send audio messages, although that should be more complex.

Thanks for your time.

**Comment by mattford** [comment]
Hey @Principe7 
There's no functionality in IRC to broadcast images/audio to channels, DCC could be used to send files directly between users, however on ScoutLink DCC is blocked by default (the receiver has to explicitly allow it ahead of time), this isn't indicated to the user so in most cases this functionality would just fail with no indication to the user.

The other option is to have the app upload the file(s) to cloud storage, e.g., an S3 bucket and share a link to the channel. However I think I'd rather not get into storing user content in a bucket on my AWS/GCP account due to problems which may arise from that, and it's easy enough to just upload images to a host yourself.

I suppose we could consider an integration with imgur or something like that, but the problem with that is it would be uploading  user (potentially children)'s pictures to the web without making that clear to them.

Basically I think it would be better to just dodge this issue entirely since there isn't a way to keep it "enclosed" within IRC.

Hope you understand, it was a good suggestion!

**Comment by Principe7** [comment]
Hello Matt

Exactly I was referring to an integration with imgur, in fact I have it integrated into my webchat as a complement to kiwiirc, and I was asking if you could do it at the app level.

Thank you anyway.
